cartoframes/data/observatory/catalog/dataset.py

```diff
@@ -97,45 +177,63 @@ def language(self):
 
     @property
     def geography(self):
-        """Id of the Geography associated to this dataset."""
+        """Id of the :obj:`Geography` associated to this dataset."""
 
         return self.data['geography_id']
 
     @property
     def geography_name(self):
+        """Name of the :obj:`Geography` associated to this dataset."""
+
         return self.data['geography_name']
 
     @property
     def geography_description(self):
+        """Description of the :obj:`Geography` associated to this dataset."""
+
         return self.data['geography_description']
 
     @property
     def temporal_aggregation(self):
-        """Time amount in which data is aggregated in this dataset."""
+        """Time amount in which data is aggregated in this dataset.
+
+        This is a free text field in this form: seconds, daily, hourly, monthly, yearly, etc.
+        """
 
         return self.data['temporal_aggregation']
 
     @property
     def time_coverage(self):
-        """Time range that covers the data of this dataset."""
+        """Time range that covers the data of this dataset.
+
+        Example: [2015-01-01,2016-01-01)
+
+        """
 
         return self.data['time_coverage']
 
     @property
     def update_frequency(self):
-        """Frequency in which the dataset is updated."""
+        """Frequency in which the dataset is updated.
+
+        Example: monthly, yearly, etc.
+        """
 
         return self.data['update_frequency']
 
     @property
     def version(self):
-        """Version info of this dataset."""
+        """Internal version info of this dataset."""
 
         return self.data['version']
 
     @property
     def is_public_data(self):
-        """True if the content of this dataset can be accessed with public credentials. False otherwise."""
+        """True if the content of this dataset can be accessed with public credentials.
+        False if it needs a subscription (AKA premium datasets).
+
+        See the subscription guide for more details about accessing premium datasets.
```

Comment:
There's no subscription guide, this will be explained in the ` Enrich your Datasets` guide. However, this is not available yet. Another option would be to link it to the Quickstart, but I don't think it's the best. I would add a link to support (I don't know which link we should use for this case cc @cmongut)

Reply:
That's one of the things I'd like to update once we have the guides. I was thinking on including a subscription guide, together with the discovery one.

I guess your point is that we don't have it yet, so we'll have the reference suggesting to see a guide that does not exist, right?

If that's the case I can remove that line by now until we complete the guides.

Reply:
Okay! Let's remove it then by now, we can add a comment in the [Enrich your Datasets](https://github.com/CartoDB/cartoframes/issues/997) issue to add it once we write it 👍